developer_copr_setup.md

```diff
@@ -0,0 +1,157 @@
+## Developer Copr Setup:
```

Comment:
Unless we're changing all links/basearchs/mock examples to 7, let's update the header to "Developer Copr Setup for CentOS 6x appliances" to minimize confusion for now.